docs/boards/asterix/index.md

```diff
@@ -32,6 +34,12 @@ Asterix and Core B2B v2
 3. Debug UART routing
 
    - L: External connector (5)
+
+     ```{warning}
+     NOTE: the UART signals are 1.8V, do not use a 3.3V adapter or there is
+     risk of damage!
+     ```
```

Comment:
```suggestion
     ```{warning}
     External connectors (SWD/UART) expose 1.8V signals.
     Do not use any adapter that does not operate at 1.8V, or there is a **risk of damage**!
     ```
```

also, move to the top (before list starts, 1. Asterix...)

Reply:
done